node/caching-compiler.js

```diff
@@ -48,7 +48,7 @@ const fallbackPkgInfo = {
   cache: Object.create(null)
 };
 
-function readWithCache(info, content) {
+function readWithCache(info, content, options) {
   const json = info && info.json;
   const reify = json && json.reify;
   const cacheFilename = getCacheFilename(content, reify);
```

Comment:
The new options should probably be included in the hash computed by `getCacheFilename`, right?